src/_11ty/filters/getArticleType.js

```diff
@@ -18,9 +18,9 @@ module.exports = tags => {
     'press-release': 'PRESS RELEASE',
   };
 
-  // tags to lowercase, strip out the first 6 chars and check includes tag suffix
+  // tags to lowercase, strip out the first 3 chars and check includes tag suffix
   const lcTags = tags
-    .map(tag => tag.toLowerCase().substr(6))
+    .map(tag => tag.toLowerCase().substr(3))
```

Comment:
Mm 🤔 Let's consider this one for a second. While we've opted for the `en` convention over `en-GB` for our initial language site(s), it's not to say the requirement for a `pt-br` or similar mightn't come up in future. Is there a different way we can split the locale off the tag? Or do we instead try to leverage our known `locale` as part of the `filter()` instead? Come at it from the other angle?

Reply:
This has all been revamped and removes any issue with the locale convention `en` over `en-GB`. It's also been enhanced to support translated values whereas before it was english only.

The translated values update spawned a tangent highlighting our search results page needed some translation too so this has been included on this branch.